ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -43,102 +62,213 @@ type OwnProps = NavigationInjectedProps<NavigationParams>;
 
 type Props = OwnProps &
   ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>;
+  ReturnType<typeof mapDispatchToProps> &
+  Readonly<{ isLoading: boolean; loadingCaption?: string | undefined; loadingOpacity?: number | undefined; onCancel?: any }>;
+
+const styles = StyleSheet.create({
+  walletBannerText: {
+    alignItems: "flex-end",
+    flexDirection: "row"
+  },
+
+  noBottomPadding: {
+    padding: variables.contentPadding,
+    paddingBottom: 30
+  },
+
+  whiteBg: {
+    backgroundColor: variables.colorWhite
+  },
+
+  brandDarkGray: {
+    color: variables.brandDarkGray
+  },
+
+  cardContainer: {
+    height: 235,
+    width: "100%",
+    position: "absolute",
+    top: 16,
+    zIndex: 7,
+    elevation: 7,
+    alignItems: "center"
+  },
+  headerSpacer: {
+    height: 172
+  },
+  cancelButton: {
+    borderColor: IOColors.red,
+    width: '90%',
+    height: variables.btnHeight + 10,
+    marginHorizontal: variables.contentPadding,
+    marginTop: 16,
+  }
+});
+
 
 const contextualHelpMarkdown: ContextualHelpPropsMarkdown = {
   title: "wallet.walletCardTransaction.contextualHelpTitle",
   body: "wallet.walletCardTransaction.contextualHelpContent"
 };
 
+const UnsubscribeButton = (props: { onPress?: () => void }) => (
+  <ButtonDefaultOpacity bordered={true} style={styles.cancelButton} onPress={props.onPress}>
+    <Label color={"red"}>{I18n.t("wallet.bancomat.details.removeCta")}</Label>
+  </ButtonDefaultOpacity>
+);
+
```

Comment:
This `UnsubscribeButton` is also used in `BancomatDetailScreen` and should be isolated in his own file instead of being copy-pasted